open-machine-learning-jupyter-book/deep-learning/nn.ipynb

```diff
@@ -524,14 +524,70 @@
     "Try the exercises about [neural networks classify 15 fruits](../../assignments/deep-learning/nn-classify-15-fruits-assignment.ipynb) and [neural networks for classification](../../assignments/deep-learning/nn-for-classification-assignment.ipynb)."
```

Comment:
<p>The output does not display the video, perhaps placing the link instead of display？</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
I don't know why it was't displayed, so I have put the link instead.